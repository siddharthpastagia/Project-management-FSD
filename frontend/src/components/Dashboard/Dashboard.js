import React from "react";
import { Jumbotron, Row, Col } from "react-bootstrap";
import "./Dashboard.scss";

export const Dashboard = () => {
  return (
    <div className="container-fluid">
      <div className="wrapper">
        <Row>
          <Col>
            <Jumbotron>
              <h1>Welcome to Project Manager App</h1>
            </Jumbotron>
          </Col>
        </Row>
      </div>
    </div>
  );
};
