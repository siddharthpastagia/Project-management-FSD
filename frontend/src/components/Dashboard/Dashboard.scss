.wrapper {
  padding: 1.5em 2em;
  float: left;
  width: 100%;
}
