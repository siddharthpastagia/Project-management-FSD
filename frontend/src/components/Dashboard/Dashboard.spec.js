import React from "react";
import { MemoryRouter as Router } from "react-router-dom";
import { render, cleanup } from "@testing-library/react";
import { Dashboard } from "./Dashboard";

describe("Dashboard Component", () => {
  afterEach(cleanup);
  it("Should render Home component", () => {
    const { container } = render(
      <Router>
        <Dashboard />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
