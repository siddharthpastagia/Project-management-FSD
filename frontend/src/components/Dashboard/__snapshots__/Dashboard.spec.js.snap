// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Component Should render Home component 1`] = `
<div>
  <div
    class="container-fluid"
  >
    <div
      class="wrapper"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <div
            class="jumbotron"
          >
            <h1>
              Welcome to Project Manager App
            </h1>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
